minecraft/auth/authclient/client.go

```
@@ -0,0 +1,118 @@
+			resp.StatusCode == 999 ||
+			(resp.StatusCode >= 500 && resp.StatusCode < 600) {
+			// Read Retry-After header before closing body
+			if resp.StatusCode == http.StatusTooManyRequests || resp.StatusCode == http.StatusServiceUnavailable {
+				if retryAfter := resp.Header.Get("Retry-After"); retryAfter != "" {
+					if seconds, parseErr := strconv.Atoi(retryAfter); parseErr == nil {
+						retryAfterDelay = time.Duration(seconds) * time.Second
+					}
+				}
+			}
+			// Always close the response body immediately
+			resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)